Meter.h

```diff
@@ -64,6 +64,7 @@ typedef struct MeterClass_ {
    const Meter_GetCaption getCaption;
    const Meter_GetUiName getUiName;
    const int defaultMode;
+   const int supportedMode;                /* bitset of supported modes, 1<<mode_id */
```

Comment:
I think the bit-field should use an `unsigned` type here. (Sorry for not informing earlier.)